src/index.css

```diff
@@ -0,0 +1,22 @@
+* * {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --font-size-nav: 25px;
+  --background-color: #ec4c8a;
+  --background-color-variant: #b03968;
+  --white: #fff;
+}
+
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
rhfhrtth

Reply:
You're right! I forgot That. It's done. Thank you @DuaneDave

Reply:
-  Well resolved

Reply:
@DuaneDave 
The part of the code that I made proud it's on the video added to my readme file. It's on the last part of the video and talks about the reload page on the details page.

Reply:
I watched the video. Great job done @ClaudiaRojasSoto 🥇